packages/api/src/routes/webhook/fitbit.ts

```diff
@@ -0,0 +1,23 @@
+import { Request, Response } from "express";
+import Router from "express-promise-router";
+import status from "http-status";
+import { processData } from "../../command/webhook/fitbit";
+import { asyncHandler } from "../util";
+
+const routes = Router();
+
+/** ---------------------------------------------------------------------------
+ * POST /webhook/fitbit
+ *
+ * Receive fitbit data for all data types for the specified user ID
+ *
+ */
+routes.post(
+  "/",
+  asyncHandler(async (req: Request, res: Response) => {
+    processData(req.body);
```

Comment:
We're sending `any` (which fits into anything) to a function that specifically requires `FitbitWebhookNotification[]`.

This should be avoided, as we don't know what we're sending, but when coding `processData` we would expect that parameter to match `FitbitWebhookNotification[]`.

In such cases, I think we should either:
1. make `processData` to receive `any`; OR
2. process `req.body` with a zod schema to validate/convert it to the expected type.

I suggest we go with (2), like we did for Apple webhooks.